masterlist.yaml

```diff
@@ -9523,30 +9523,50 @@ plugins:
         name: 'SM DLC Plugin Refurbish'
     group: *veryEarlyGroup
   - name: 'SM Plugin Refurbish Lite.esp'
-    after:
-      - 'Knights - Unofficial Patch.esp'
-      - 'Knights Infamy.esp'
+    req: [ 'DLCShiveringIsles.esp' ]
     msg:
       - *alsoUseSilentVoice
+      - <<: *patchIncluded
+        subs: [ 'Knights of the Nine - Improved Infamy System' ]
+        condition: 'active("Knights Infamy.esp") and not file("SM Plugin Refurbish Lite Knights Infamy.esp")'
       - <<: *sortMasters
-        condition: 'checksum("SM Plugin Refurbish Lite.esp", 0466DC3B)'
+        condition: 'checksum("SM Plugin Refurbish Lite.esp", F05C7945)'
     tag: [ Actors.RecordFlags ]
     clean:
```

Comment:
You can omit clean checksums for mods whose masters can be sorted for the same reasons you can do so for dirty plugins:

```suggestion
```